### DISCUSSION TITLE
FLSUN SR extruder calibration issues

### DISCUSSION THREAD

**Discussion by dimitrisgia** [blackfyre/flsun.community#47] [Q&A] [answered]
I have been using this printer for a few months. All was going well until I upgraded the extruder, never finished good prints afterward even after calibrating the extruder, and then reinstalled the original extruder.
I then upgraded the firmware, reset to factory defaults and tried to calibrate the extruder again, but now it will no longer accept or change the esteps for the extruder.
I placed a .gcode document on the sd card with the commands:
M92 E390 (or any number really)
M500
Before I updated the firmware, and I ran it in the printer, it would update the amount extruded. But now after the firmware update, the printer will no longer respond to ANY changes and only extrudes 80mm instead of 100mm.
I cannot get it to change no matter what numbers I put in,
Anyone know how to help me calibrate this extruder or what I am doing wrong?

Feeling pretty dumb...

**Comment by dimitrisgia** [comment, marked as answer]
I found the solution. Thank you anyway

**Reply by karmavibes7** [reply]
hi, can you tell us the solution please ? i have same problem but i found nothing... same like you, after a firmware upgrade, M92 don't seem to work with the new value when i try to re-check the values.
